src/css/index.scss

```diff
@@ -0,0 +1,230 @@
+* {
+  box-sizing: border-box;
```

Comment:
Good to reset all the tags to the border-box for the consistency across the browsers 👍